pandas/tests/window/test_apply.py

```diff
@@ -44,7 +44,7 @@ def f(x):
     expected = df.iloc[2:].reindex_like(df)
     tm.assert_frame_equal(result, expected)
 
-    with pytest.raises(AttributeError):
+    with tm.external_error_raised(AttributeError):
```

Comment:
what is this error?

Reply:
`E       AttributeError: 'numpy.ndarray' object has no attribute 'index'` in both cases

Reply:
I suppose from a numpy array not having an `index` attribute

Reply:
hmm. i think we could raise a better message here as to what the issue is (trying to use pandas functions inside the rolling when raw=True). cc @mroeschke let me create an issue.

Reply:
https://github.com/pandas-dev/pandas/issues/38809